styles.css

```diff
@@ -0,0 +1,523 @@
+    --roze: #ED4385;
+    --zwart: #0C0C0C;
+    --grijs: #7A7A7A;
+
+}
+*{  /*reset css*/
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    overflow-x: hidden;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+*{
+    font-family: roboto,sans-serif;
+    font-style: normal;
+}
+h1,h2,h3,h4,h5,h6{
+    font-family: 'Merriweather', serif;
+}
+h1{
+    font-size:2.4rem;
+}
+.small-weight-p{
+    font-weight: 300;
+}
+a{
+    text-decoration: none;
+    color: var(--paars);
+}
+a:hover{
+    color: rgba(87, 34, 205, 0.7);
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+/* Header {Nav, Hero}*/
+header {
+    width: 101%;
+    height: 560px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+nav{
+    width: 100%;
+    height: 70px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-end;
+    margin-right: 80px;
+    margin-top: 25px;
+}
+nav ul{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    gap: 3rem;
+    list-style: none;
+}
+nav ul li a{
+    text-decoration: none;
+    color: white;
+    font-weight: 300;
+}
+nav ul li a:hover{
+    color: rgba(255, 255, 255, 0.7);
+}
+nav ul li a:visited{
+    color: white;
+}
+
+header h1,header p{
+    color: white;
+    margin-bottom: 30px;
+}
+header p{
+    font-size: 1.6rem;
+}
+header div{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 70px;
+    text-align: center;
+}
+header img{
+    justify-self: center;
+    margin-top: 120px;
+}
+/*Main */
+main{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 101%;
+}
+/*introdution section */
+
+
+.section-intro{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 100px 0;
+    position: relative;
+}
+
+.section-intro div{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.section-intro div h2{
+    text-align: center;
+    color: var(--paars);
+    padding-bottom: 30px;
+}
+.section-intro div p{
+    padding-top: 70px;
+}
+/*info-blocks section */
+.section-info-blocks{
+    position: relative;
+    background-color: var(--lichtgrijs);
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 180px 20px 60px 20px;
+    z-index: 0;
+
+}
+.info-block{
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 60px;
+
+    align-items: center;
+    justify-content: space-between;
+    z-index: 2;
+
+}
+.info-block:nth-child(even){
+    flex-direction: row-reverse;
+}
+
+.section-info-blocks div article div{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+    max-width: 300px;
```

Comment:
Omdat deze div in een `info-block` staat die als container een row-richting heeft, kun je hier gewoon `flex-basis` gebruiken en dan `flex-grow` op 0 zetten :)